client/a8c-for-agencies/sections/sites/features/jetpack/jetpack-preview-pane.tsx

```diff
@@ -0,0 +1,129 @@
+import { useTranslate } from 'i18n-calypso';
+import React, { useCallback, useContext, useEffect, useMemo } from 'react';
+import {
+	JETPACK_ACTIVITY_ID,
+	JETPACK_BACKUP_ID,
+	JETPACK_BOOST_ID,
+	JETPACK_MONITOR_ID,
+	JETPACK_PLUGINS_ID,
+	JETPACK_SCAN_ID,
+	JETPACK_STATS_ID,
+} from 'calypso/a8c-for-agencies/sections/sites/features/features';
+import SitesDashboardContext from 'calypso/a8c-for-agencies/sections/sites/sites-dashboard-context';
+import { useJetpackAgencyDashboardRecordTrackEvent } from 'calypso/jetpack-cloud/sections/agency-dashboard/hooks';
+import SitePreviewPane, { createFeaturePreview } from '../../site-preview-pane';
+import { PreviewPaneProps } from '../../site-preview-pane/types';
+import { JetpackActivityPreview } from './activity';
+import { JetpackBackupPreview } from './jetpack-backup';
+import { JetpackBoostPreview } from './jetpack-boost';
+import { JetpackMonitorPreview } from './jetpack-monitor';
+import { JetpackPluginsPreview } from './jetpack-plugins';
+import { JetpackScanPreview } from './jetpack-scan';
+import { JetpackStatsPreview } from './jetpack-stats';
+
+export function JetpackPreviewPane( {
+	site,
+	closeSitePreviewPane,
+	className,
+	isSmallScreen = false,
+	hasError = false,
+}: PreviewPaneProps ) {
+	const translate = useTranslate();
+	const recordEvent = useJetpackAgencyDashboardRecordTrackEvent( [ site ], ! isSmallScreen );
```

Comment:
Do we want to be copying over the 'useJetpackAgencyDashboardRecordTrackEvent' hook? It creates an event name beginning with `calypso_jetpack_agency_dashboard_` which doesn't seem relevant here.
Or would this be considered part of a later refactor?

Reply:
Mmmm, Good catch!

I think if we spot them, we should remove  on the fly and later add the correct tracked events.

Reply:
Created an issue so that we can copy over and change the hook itself: https://github.com/Automattic/automattic-for-agencies-dev/issues/138